tests/Maester/Azure/Test-MtUserAccessAdmin.md

```diff
@@ -0,0 +1,28 @@
+Ensure that no person has permanent access to Azure Subscriptions.
+
+User Access Administrator is a role that allows an Administrator to perform everything on an Azure Subscription. Global Administrators can gain this permission on the Root Scope in Entra ID, in the properties of Entra ID. These permissions should only be used in case of emergency and should not be assigned permanently.
+
+Ensure that no User Access Administrator permissions at the Root Scope are applied.
+
+#### Remediation action:
+
+To remove all Admins with Root Scope permissions, as a Global Admin:
+1. Navigate to Microsoft 365 admin center [https://portal.microsoft.com](https://portal.microsoft.com).
+2. Search for **Microsoft Entra ID** select **Microsoft Entra ID**.
+3. Expand the **Manage** menu, select **Properties**
```

Comment:
Replace the comma with the word "and" and end the sentence with a period.

"Expand ... and select ..."